src/components/HeaderWithBackButton/index.js

```diff
@@ -49,10 +52,12 @@ function HeaderWithBackButton({
     shouldEnableDetailPageNavigation = false,
     children = null,
     shouldOverlay = false,
+    singleExecution = (func) => func,
 }) {
     const [isDownloadButtonActive, temporarilyDisableDownloadButton] = useThrottledButtonState();
     const {translate} = useLocalize();
     const {isKeyboardShown} = useKeyboardState();
+    const waitForNavigate = useWaitForNavigation();
```

Comment:
👋 coming from https://github.com/Expensify/App/issues/30636

The hooks here cause an error in the storybook for component `HeaderWithBackButton`.